versions/1/manifest.spec.yml

```diff
@@ -273,6 +273,25 @@ spec:
           pattern: '^(([a-zA-Z0-9-]+)|([a-zA-Z0-9-]+\/[a-zA-Z0-9-]+))$'
       required:
       - github
+    buildDependencies:
+      type: array
+      description: Build-time dependencies
+      items:
+        type: object
+        additionalProperties: false
+        properties:
+          name:
+            type: string
+            description: Name of the dependency
+            enum:
+              - ecs
+          reference:
+            type: string
+            description: Source reference
+            pattern: 'git@[a-z0-9]{40}'
```

Comment:
Allow the use of tags too? (Can be extended in the future in any case)

Reply:
The reason why I chose a commit hash is because it's always the same for all develops. If we put a tag here (e.g. `git@1.9.0`), the elastic-package will build a slightly different package based on the commit marked with the tag. I was thinking about a switch for `elastic-package build`, for example: `--update`, which can update references to latest dependency versions.

Actually I see a problem with this design: how can I pick the latest commit for ECS 1.9? Let me suggest something.

Reply:
Ok, I adjusted the manifest to include additional field "version". The update procedure would be as follows:

1. Check ECS dependency considering version (e.g. `1.9`).
2. Fetch latest commit from the branch `1.9`.
3. Update `reference` with the commit hash.

Reply:
@mtojek having a `version` and a `reference` can be confusing, even contradictory if someone sets a reference that doesn't correspond to a `version`. I think that we can go on by now with your original approach with the `reference` only, and we can improve it later to support other use cases.

> how can I pick the latest commit for ECS 1.9? 

> Fetch latest commit from the branch `1.9`.

I wouldn't try to support branches, at least by now, configured branches may lead to unreproducible builds, same code build at different moments can result in packages with different fields.

Reply:
++

Reply:
Fixed